app/src/components/header/Header.vue

```diff
@@ -42,18 +42,18 @@ import Vue from 'vue'
   }
 })
 export default class Header extends Vue {
-  showMenu = false
-  userInfo = {
+  showMenu: boolean = false
+  userInfo: { name: string; avatar: string } = {
     name: 'John Doe',
     avatar: 'Man.jpg'
   }
-  menuItems = [
+  menuItems: { id: number; route: string; title: string }[] = [
```

Comment:
Maybe it makes sense to add an interface for the `menuItems`.

Reply:
Thanks, added this interface in the new commit.